internal/config/config.go

```diff
@@ -47,9 +47,10 @@ type Config struct {
 	InitContainerImages   []string          `desc:"List of init containers that should be appended for each deployment that has Config.Annotation" split_words:"true"`
 	ContainerImages       []string          `desc:"List of containers that should be appended for each deployment that has Config.Annotation" split_words:"true"`
 	Envs                  []string          `desc:"Additional Envs that should be appended for each Config.ContainerImages and Config.InitContainerImages" split_words:"true"`
-	CertFilePath          string            `desc:"Path to certificate" split_words:"true"`
-	KeyFilePath           string            `desc:"Path to RSA/Ed25519 related to Config.CertFilePath" split_words:"true"`
-	CABundleFilePath      string            `desc:"Path to cabundle file related to Config.CertFilePath" split_words:"true"`
+	WebhookMode           Mode              `default:"0" desc:"Default '0' mode uses spire certificates and external webhook configuration. Set to '1' to use the automatically generated webhook configuration" split_words:"true"`
```

Comment:
I'm not familiar with the config parsing lib.
Is it possible to use words instead of numbers for webhook mode?

Reply:
Good idea!
Fixed